cert.go: warning: control reaches end of non-void function

``Set_meta`` is [defined as returning](https://github.com/zeromq/goczmq/blob/master/cert.go#L6) ``int`` in ``cert.go``:

````
int Set_meta(zcert_t *self, const char *key, const char *value) {zcert_set_meta(self, key, value);}                                           
````

This results in a warning in an otherwise clean compilation.

````
vendor/github.com/zeromq/goczmq/cert.go:6:99: warning: control reaches end of non-void function [-Wreturn-type]
````

The only thing Set_meta does is call [``zcert_set_meta``](http://czmq.zeromq.org/manual:zcert):
````
//  Set certificate metadata from formatted string.
CZMQ_EXPORT void
    zcert_set_meta (zcert_t *self, const char *name, const char *format, ...);
````

Redefining ``Set_meta`` with a void return seems reasonable.

--- a/AUTHORS
+++ b/AUTHORS
@@ -7,3 +7,4 @@ Indradhanush Gupta
 Armen Baghumian
 Ben Aldrich
 Jordan Sissel
+Nick Bargnesi

--- a/cert.go
+++ b/cert.go
@@ -3,7 +3,7 @@ package goczmq
 /*
 #include "czmq.h"
 
-int Set_meta(zcert_t *self, const char *key, const char *value) {zcert_set_meta(self, key, value);}
+void Set_meta(zcert_t *self, const char *key, const char *value) {zcert_set_meta(self, key, value);}
 */
 import "C"
 

